torchvision/transforms/functional.py

```diff
@@ -521,7 +521,12 @@ def perspective(
     coeffs = _get_perspective_coeffs(startpoints, endpoints)
 
     if not isinstance(img, torch.Tensor):
-        return F_pil.perspective(img, coeffs, interpolation=interpolation, fill=fill)
+        np_endpoints = np.array(endpoints)
+        new_size = tuple([
+            np.max(np_endpoints[:, 0]) - np.min(np_endpoints[:, 0]) + 1,
+            np.max(np_endpoints[:, 1]) - np.min(np_endpoints[:, 1]) + 1])
```

Comment:
can we use torch tensors here instead of `numpy` arrays? We try to depend on numpy only for conversion functions in this file.

Reply:
OK, let me modify that

Reply:
done